package.json

```diff
@@ -7,21 +7,21 @@
   "license": "MIT",
   "dependencies": {
     "path-root": "^0.1.1",
-    "resolve": "^1.10.0"
+    "resolve": "^1.13.1"
   },
   "devDependencies": {
-    "@types/chai": "^4.1.7",
-    "@types/fs-extra": "^5.0.5",
-    "@types/mocha": "^5.2.6",
-    "@types/node": "^11.10.4",
-    "@types/semver": "^5.5.0",
+    "@types/chai": "^4.2.6",
+    "@types/fs-extra": "^8.0.1",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.12.14",
+    "@types/semver": "^6.2.0",
     "chai": "^4.2.0",
-    "execa": "^1.0.0",
-    "fixturify-project": "^1.7.0",
-    "fs-extra": "^7.0.1",
-    "mocha": "^6.0.2",
-    "semver": "^5.6.0",
-    "typescript": "^3.3.3333"
+    "execa": "^3.4.0",
```

Comment:
CI failure is due to this bump (or at least this is the first one that causes it):

```
$ yarn
yarn install v1.13.0
[1/4] Resolving packages...
[2/4] Fetching packages...
error execa@3.4.0: The engine "node" is incompatible with this module. Expected version "^8.12.0 || >=9.7.0". Got "6.17.1"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```